### PR TITLE
[For 3.0.0] Feature/2943 add username in AdminUser/Account info

### PR DESCRIPTION
**Pull request by metzgegu**
#2943 

## Checkpoints

The goal of this section is to help code reviewer and merge request author to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

**For code reviewer**

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

**For developper**

- [x] Manual tests done to ensure stability on whole application layers, issue expectation follow
- [x] Original issue have been updated with a short resume of implemented solution

**For quality team**

- [x] Tested by quality team

### CODE REVIEW

File: frontend/src/component/Account/PersonalData.jsx [hunk 1]
```diff
@@ -26,13 +32,14 @@ export class PersonalData extends React.Component {
   handleClickSubmit = async () => {
     const { props, state } = this
 
-    if (state.newEmail !== '' && state.checkPassword === '') {
+    if ((state.newEmail !== '' || state.newUsername !== '') && state.checkPassword === '') {
       props.dispatch(newFlashMessage(props.t('Please type your password in order to change your email. (For security reasons)'), 'warning'))
```
Comment (raphj) [inline frontend/src/component/Account/PersonalData.jsx:36]: It looks like this message should probably be updated

**Comment by raphj** [comment]
After tests:

- we are missing the list of authorized characters (adding this to the ticket with @celiamgtt's approval)
- double `@@` when typing a new username with the `@` prefix as per this video:
  atat
 - the warning string should be updated after the related decision is taken in the ticket

File: frontend/i18next.scanner/fr/translation.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
+  "Allowed characters: {{allowedCharactersUsername}}": "Des caractères autorisés : {{allowedCharactersUsername}}",
```
Comment (raphj) [inline frontend/i18next.scanner/fr/translation.json:2]:
```suggestion
  "Allowed characters: {{allowedCharactersUsername}}": "Caractères autorisés : {{allowedCharactersUsername}}",
```

**Comment by inkhey** [comment]
verified on branch

**Pull request merged**